vuepress/docs/next/docs/consume/custom-resources.md

```diff
@@ -355,7 +289,7 @@ metadata:
   name: "test-plugin"
   namespace: "my-namespace"
 spec:
-  image: your-org/your-image:4.3.2
+  image: your-org/your-image
```

Comment:
This version still needs to be here. When you specify a Docker/Linux image you have to indicate the user or organization (your-org), the name of image (your-image), and the version of the image (4.3.2, it could also be a tag like latest but a specific version is considered safer.)